### DISCUSSION TITLE
How to use on Windows?

### DISCUSSION THREAD

**Discussion by bisoloc** [d99kris/nchat#181] [Q&A]
Hey, how can I use this on Windows?

**Comment by d99kris** [comment]
Hi @bisoloc - you can [install WSL](https://learn.microsoft.com/en-us/windows/wsl/install) (which effectively is a Linux virtual machine, based on Ubuntu by default). Then start `Ubuntu` from the start menu, and follow the instructions under https://github.com/d99kris/nchat#build-from-source

I occassionally run nchat via WSL and I have not encountered any issues.

Building and running nchat as a native Windows `.exe` executable is not officially supported, so I recommend using WSL.

**Reply by bisoloc** [reply]
Hello @d99kris,

I'm sorry for the late reply. I just read on my Gmail that I received a notification from GitHub where you replied. Thank you so much for taking the time to reply and guide me properly. I'm installing WSL/Ubuntu on Windows and then I'll try nchat.
